tools/packaging/kernel/build-kernel.sh

```diff
@@ -199,10 +199,7 @@ get_kernel_frag_path() {
 	# Later, if we need to add kernel version specific subdirs in order to
 	# handle specific cases, then add the path definition and search/list/cat
 	# here.
-	local all_configs="${common_configs} ${arch_configs}"
-	if [[ ${experimental_kernel} == "true" ]]; then
-		all_configs="${all_configs} ${experimental_configs}"
-	fi
+	local all_configs="${common_configs} ${arch_configs} ${experimental_configs}"
```

Comment:
I *think* we should probably just move virtio-fs options from experimental to the appropriate file in common? that way we have the pattern for future 'experimental' features?

Reply:
I'll try to forward port the change I made that does this in 1.x

Reply:
See: https://github.com/kata-containers/kata-containers/pull/962